### ISSUE TITLE
Long query makes the text messy

### ISSUE THREAD

**Issue by ArttuLai** [Flow-Launcher/Flow.Launcher#153]
## Describe the Bug
If I write a long query using the default Web Searches plugin, the text will mess up and overlap with itself. This happens when the length of the text you're typing extends the width of the Flow window, when you're using an action keyword that activates a search engine. So this doesn't happen if you just type a long query without activating the Web Searches plugin.

## To Reproduce
1. Enable Web Searches plugin via Flow's settings.
2. Open Flow with the hotkey and type a long query using one of the search engines, like Google.
3. When the width of the paragraph you're writing extends the width of Flow bar, the text starts to double / overlap / mess up. 

Note that this doesn't happen if you just type the text without using an action keyword for a search engine.

## Screenshots
<img width="622" alt="longTextMessy" src="https://user-images.githubusercontent.com/42007282/92303954-6fe09e80-ef82-11ea-857c-c5af15438a2e.png">

### Your System
```
Windows build number: Windows 10 20H2, 19042.487
Flow Launcher version: 1.2.1
```
### Flow Launcher Error Log
[2020-09-05.txt](https://github.com/Flow-Launcher/Flow.Launcher/files/5177964/2020-09-05.txt)

**Comment by jjw24** [comment]
discussion: https://github.com/Flow-Launcher/Flow.Launcher/discussions/736

**Comment by Garulf** [comment]
Fixed in 1.9.0 release with https://github.com/Flow-Launcher/Flow.Launcher/pull/860
